jobs/ci-kubernetes-node-docker-benchmark.sh

```diff
@@ -19,4 +19,23 @@ set -o pipefail
 set -o xtrace
 
 # PROJECT="k8s-jkns-ci-node-e2e"
-test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/docker_validation/jenkins-perf.properties
+readonly testinfra="$(dirname "${0}")/.."
+
+export NODE_IMG_VERSION="master"
+export NODE_TEST_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
+export NODE_TEST_PROPERTIES="test/e2e_node/jenkins/docker_validation/jenkins-perf.properties"
+
+### Runner
+readonly runner="${testinfra}/jenkins/node-dockerized.sh"
+export KUBEKINS_TIMEOUT="90m"
+timeout -k 15m "${KUBEKINS_TIMEOUT}" "${runner}" && rc=$? || rc=$?
+
+### Reporting
+if [[ ${rc} -eq 124 || ${rc} -eq 137 ]]; then
+    echo "Build timed out" >&2
+elif [[ ${rc} -ne 0 ]]; then
+    echo "Build failed" >&2
+fi
+echo "Exiting with code: ${rc}"
+exit ${rc}
+
```

Comment:
nit: some of these files have an extra trailing blank line that should probably be removed.